modules/ROOT/pages/backup-restore/restore-dump.adoc

```diff
@@ -76,6 +76,7 @@ Can contain `*` and `?` for globbing.
 
 |--from-path=<path>
 |Path to a directory containing archive(s) created with the dump command.
+As of Neo4j 5.19, archives can also be loaded from S3 URIs.  
```

Comment:
same here

`neo4j-admin database load --from-path s3://some-bucket/some/directory/ neo4j`

Reply:
added a note, an example and included info about AWS default credentials